sdk/search/generate.ps1

```diff
@@ -0,0 +1,21 @@
+#
+# Generate Azure.AI.Search
+#
+Push-Location $PSScriptRoot/Azure.AI.Search/src/
+
+# We're doing a copy instead of a reference until we can merge them together
+# because AutoRest doesn't play well with two remote swagger files.  (We only
+# warn on failure so offline regeneration plays nicely for those of us coding
+# on the bus.)
+Write-Output "Copying latest swagger files..."
+Invoke-WebRequest https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/search/data-plane/Microsoft.Azure.Search.Data/stable/2019-05-06/searchindex.json `
+    -OutFile ./swagger/searchindex.json `
```

Comment:
Should these be in a local .gitignore file (i.e. in this directory) or do we want to check them in?

Reply:
I didn't really mind checking them in for the short term.  I can also remove them if folks feel strongly.

Reply:
Nope. Was just calling it out in case it was unintentional.